sanic/app.py

```diff
@@ -219,9 +225,7 @@ def response(handler):
         return response
 
     # Shorthand method decorators
-    def get(
-        self, uri, host=None, strict_slashes=None, version=None, name=None
-    ):
+    def get(self, uri, host=None, strict_slashes=None, version=None, name=None):
```

Comment:
Looks like quite a lot of the changes in this file are formatting changes, was that just a result of merging in latest master? Or should these formatting changes be filtered out?